src/swsssdk/configdb.py

```diff
@@ -99,4 +153,6 @@ def get_config(self):
                 data[table_name] = {}
             data[table_name][key] = client.hgetall(_hash)
         return data
+        self.pubsub = self.redis_clients[self.CONFIG_DB].pubsub()
+        self.pubsub.psubscribe("__keyspace@{}__:*".format(self.db_map[self.CONFIG_DB]['db']))
```

Comment:
Deleted.

---
In reply to: [128202084](https://github.com/Azure/sonic-py-swsssdk/pull/8#discussion_r128202084) [](ancestors = 128202084)